panel/widgets/examples.py

```diff
@@ -0,0 +1,295 @@
+import os
+import re
```

Comment:
Gradio also displayes the examples in a table. I display using Panes and in a Column. Should we display using Tabulator instead. The pro of not using Tabulator is that we can display everything. The cons is that we might need to instantiate a large number of Bokeh models which is slow. And I don't know if we can make it look pretty.

## Gradio

![image](https://github.com/user-attachments/assets/7f04bda0-dff3-49ab-98a0-37bd17012d3a)


```python
import gradio as gr
from PIL import Image
import pandas as pd

# Example Data
image_example = Image.new('RGB', (100, 100), color = 'red')  # Red square image

examples = [
    ["Hello, World!", 42, 3.14, image_example, pd.DataFrame({"x": [1,2], "y": [1,2]})],
    ["Another text", 7, 0.001, image_example, pd.DataFrame({"x": [3,4], "y": [1,2]},)]
]

with gr.Blocks() as demo:
    gr.Examples(
        examples=examples,
        inputs=[gr.Textbox(), gr.Number(), gr.Number(), gr.Image(), gr.DataFrame()],
    )

demo.launch()
```